ui/src/app/views/account-creation-success.tsx

```diff
@@ -19,6 +19,14 @@ const styles = {
     outlineColor: 'transparent',
     color: colors.accent,
     backgroundColor: 'transparent',
+  },
+  borderStyle: {
+    border: '1px solid ' + colors.primary,
```

Comment:
Nit: You can do this with {backtick}1px solid ${colors.primary}{backtick} instead of string concatenation.